File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
    int height;
    int spaces;
    int hashes;
    
    do{
        printf("Height: ");
        height = get_int();
    } while(height < 0 || height > 23);
    
    for(int i = 0; i < height; i++) {
        
        for(spaces = height - i; spaces > 1; spaces--) {
            printf(" ");
        }
        
        for(hashes = 0; hashes <= i + 1; hashes++) {
            printf("#");
        }
        
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

while True:
    print("Height: ", end="")
    height = cs50.get_int()
    if height > 0 and height < 24:
        break;

spaces = height - 1
hashes = 2

for i in range(height):
    print (" " * spaces, end="")
    print ("#" * hashes)
    hashes = hashes + 1
    spaces = spaces - 1


